_posts/2020-06-30-gf-transparency-report.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "General Fund Transparency Report (2019 - June 2020)"
+summary: A report on the usage of XMR in the General Fund managed by the Core Team. Covers the years 2019 and the first half of 2020.
+tags:  core

```

Comment:
The tags should be between square brackets. Also please add `community`, so that the post will show up in the correct category. This is important all the post will show up only in the 'all blog posts' section.

The result should be:

`tags:  [community, core]`

Reply:
thanks, fixed